### ISSUE TITLE
Sparta Theme minimum contrast adjustments

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13498]
**Describe the bug**

- Automated test detects contrast below standard.
- Cart page seems to gather most of the contrast issues, so this is a good page for testing.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**
1. Add a couple of items to the cart and go to Cart page.
2. Inspect reported elements (screenshot below) 
3. Go to the "Computed" tab (next to "Styles") to confirm font-size
4. Use an online WCAG Contrast Checker to verify current contrast.

**Expected behavior**

- Pass automated test.
- Text less than 18 points or bold text less than 14 points must have a luminosity contrast ratio of **4.5:1 or more**.
- Text 18 point or larger or bold text 14 point or larger must have a luminosity contrast ratio of **3:1 or more**.
- When color alone is used to indicate links from non-linked text - the color difference between the surrounding text must meet these contrast requirements.
- It seems the automated validator takes 18pt sharp under the "less than 18pt" threshold, therefore any text of 18pt or less should have a contrast ratio of **4.5:1 or more**.

**Screenshots**
![Screen Shot 2021-08-05 at 6 30 57 PM](https://user-images.githubusercontent.com/15113219/129987774-b960f2e6-a628-4374-81fc-565f93e13b6b.png)
![Screen Shot 2021-08-05 at 6 35 30 PM](https://user-images.githubusercontent.com/15113219/129987776-c74605db-5973-4200-b5ea-701a49514472.png)
![Screen Shot 2021-08-18 at 8 03 03 PM](https://user-images.githubusercontent.com/15113219/129987825-d00d6bf5-f918-4788-a21d-e49b841aaf7a.png)

**Comment by saad-mo** [comment]
mostly styling issues. assigning to @morganm58
